package.json

```diff
@@ -60,6 +60,7 @@
     "bourbon-neat": "^1.7.4",
     "del": "^2.2.2",
     "dotenv": "^2.0.0",
+    "emailjs": "^1.0.8",
```

Comment:
Please update the shrinkwrap file too.

```bash
$ npm install
$ npm shrinkwrap
```

Then add it to git and commit.

**Note**: If shrinkwrap fails due to extraneous packages, run `npm prune` before to clean up your node_modules (things like this happen due to outdated shrinkwraps in dependencies and stuff like this).